<?xml version="1.0" encoding="UTF-8"?>
<ui version="4.0">
 <class>NewConnectionDialog</class>
 <widget class="QDialog" name="NewConnectionDialog">
  <property name="geometry">
   <rect>
    <x>0</x>
    <y>0</y>
    <width>610</width>
    <height>554</height>
   </rect>
  </property>
  <property name="windowTitle">
   <string>New Connection</string>
  </property>
  <layout class="QVBoxLayout" name="verticalLayout">
   <item>
    <layout class="QHBoxLayout" name="horizontalLayout_3">
     <item>
      <widget class="QComboBox" name="comboBox_Preset">
       <property name="sizePolicy">
        <sizepolicy hsizetype="Expanding" vsizetype="Minimum">
         <horstretch>0</horstretch>
         <verstretch>0</verstretch>
        </sizepolicy>
       </property>
      </widget>
     </item>
     <item>
      <widget class="QToolButton" name="toolButton_PresetNew">
       <property name="text">
        <string>...</string>
       </property>
       <property name="icon">
        <iconset resource="../rc/rc.qrc">
         <normaloff>:/icons/document.png</normaloff>:/icons/document.png</iconset>
       </property>
      </widget>
     </item>
     <item>
      <widget class="QToolButton" name="toolButton_PresetSave">
       <property name="text">
        <string>...</string>
       </property>
       <property name="icon">
        <iconset resource="../rc/rc.qrc">
         <normaloff>:/icons/disk_blue.png</normaloff>:/icons/disk_blue.png</iconset>
       </property>
      </widget>
     </item>
     <item>
      <widget class="Line" name="line">
       <property name="orientation">
        <enum>Qt::Vertical</enum>
       </property>
      </widget>
     </item>
     <item>
      <widget class="QToolButton" name="toolButton_PresetOpen">
       <property name="text">
        <string>...</string>
       </property>
       <property name="icon">
        <iconset resource="../rc/rc.qrc">
         <normaloff>:/icons/folder.png</normaloff>:/icons/folder.png</iconset>
       </property>
      </widget>
     </item>
     <item>
      <widget class="QToolButton" name="toolButton_PresetSaveAs">
       <property name="text">
        <string>...</string>
       </property>
       <property name="icon">
        <iconset resource="../rc/rc.qrc">
         <normaloff>:/icons/save_as.png</normaloff>:/icons/save_as.png</iconset>
       </property>
      </widget>
     </item>
    </layout>
   </item>
   <item>
    <widget class="QTabWidget" name="tabWidget">
     <property name="currentIndex">
      <number>0</number>
     </property>
     <widget class="QWidget" name="tabBasic">
      <attribute name="title">
       <string>Basic</string>
      </attribute>
      <layout class="QVBoxLayout" name="verticalLayout_3">
       <item>
        <layout class="QGridLayout" name="gridLayout">
         <item row="1" column="1">
          <layout class="QHBoxLayout" name="horizontalLayout_2">
           <item>
            <widget class="QLineEdit" name="lineEdit_Host">
             <property name="placeholderText">
              <string>Type here the server name</string>
             </property>
            </widget>
           </item>
           <item>
            <widget class="QSpinBox" name="spinBox_Port">
             <property name="minimumSize">
              <size>
               <width>80</width>
               <height>0</height>
              </size>
             </property>
             <property name="maximum">
              <number>65535</number>
             </property>
             <property name="value">
              <number>18000</number>
             </property>
            </widget>
           </item>
          </layout>
         </item>
         <item row="1" column="0">
          <widget class="QLabel" name="label">
           <property name="text">
            <string>Host</string>
           </property>
          </widget>
         </item>
         <item row="0" column="0">
          <widget class="QLabel" name="label_2">
           <property name="text">
            <string>Name</string>
           </property>
          </widget>
         </item>
         <item row="0" column="1">
          <widget class="QLineEdit" name="lineEdit_Name">
           <property name="placeholderText">
            <string>Type here connection name</string>
           </property>
          </widget>
         </item>
        </layout>
       </item>
       <item>
        <widget class="CStreamListWidget" name="streamListWidget" native="true"/>
       </item>
      </layout>
     </widget>
     <widget class="QWidget" name="tabAdvanced">
      <attribute name="title">
       <string>Advanced</string>
      </attribute>
     </widget>
     <widget class="QWidget" name="tabDescription">
      <attribute name="title">
       <string>Description</string>
      </attribute>
      <layout class="QVBoxLayout" name="verticalLayout_2">
       <item>
        <widget class="QPlainTextEdit" name="plainTextEdit_Description"/>
       </item>
       <item>
        <widget class="QCheckBox" name="checkBox_WrapMode">
         <property name="text">
          <string>Enable Wrap Mode</string>
         </property>
        </widget>
       </item>
      </layout>
     </widget>
    </widget>
   </item>
   <item>
    <layout class="QHBoxLayout" name="horizontalLayout">
     <item>
      <spacer name="horizontalSpacer">
       <property name="orientation">
        <enum>Qt::Horizontal</enum>
       </property>
       <property name="sizeHint" stdset="0">
        <size>
         <width>40</width>
         <height>20</height>
        </size>
       </property>
      </spacer>
     </item>
     <item>
      <widget class="QPushButton" name="pushButton">
       <property name="text">
        <string>Connect</string>
       </property>
       <property name="default">
        <bool>true</bool>
       </property>
      </widget>
     </item>
     <item>
      <widget class="QPushButton" name="pushButton_3">
       <property name="text">
        <string>Close</string>
       </property>
      </widget>
     </item>
    </layout>
   </item>
  </layout>
 </widget>
 <customwidgets>
  <customwidget>
   <class>CStreamListWidget</class>
   <extends>QWidget</extends>
   <header location="global">cstreamlistwidget.h</header>
   <container>1</container>
  </customwidget>
 </customwidgets>
 <tabstops>
  <tabstop>toolButton_PresetNew</tabstop>
  <tabstop>toolButton_PresetSave</tabstop>
  <tabstop>toolButton_PresetOpen</tabstop>
  <tabstop>toolButton_PresetSaveAs</tabstop>
  <tabstop>lineEdit_Name</tabstop>
  <tabstop>lineEdit_Host</tabstop>
  <tabstop>spinBox_Port</tabstop>
  <tabstop>checkBox_WrapMode</tabstop>
  <tabstop>plainTextEdit_Description</tabstop>
  <tabstop>tabWidget</tabstop>
  <tabstop>pushButton</tabstop>
  <tabstop>pushButton_3</tabstop>
  <tabstop>comboBox_Preset</tabstop>
 </tabstops>
 <resources>
  <include location="../rc/rc.qrc"/>
 </resources>
 <connections>
  <connection>
   <sender>pushButton</sender>
   <signal>clicked()</signal>
   <receiver>NewConnectionDialog</receiver>
   <slot>accept()</slot>
   <hints>
    <hint type="sourcelabel">
     <x>471</x>
     <y>524</y>
    </hint>
    <hint type="destinationlabel">
     <x>356</x>
     <y>523</y>
    </hint>
   </hints>
  </connection>
  <connection>
   <sender>pushButton_3</sender>
   <signal>clicked()</signal>
   <receiver>NewConnectionDialog</receiver>
   <slot>reject()</slot>
   <hints>
    <hint type="sourcelabel">
     <x>551</x>
     <y>532</y>
    </hint>
    <hint type="destinationlabel">
     <x>384</x>
     <y>532</y>
    </hint>
   </hints>
  </connection>
 </connections>
</ui>
